Build.ps1

```diff
@@ -1,85 +1,15 @@
-param(
-    [String] $majorMinor = "0.0",  # 2.0
-    [String] $patch = "0",         # $env:APPVEYOR_BUILD_VERSION
-    [String] $customLogger = "",   # C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll
-    [Switch] $notouch,
-    [String] $sln                  # e.g serilog-sink-name
-)
+Push-Location $PSScriptRoot
```

Comment:
This script is slightly older than the one in serilog/serilog (there were some issues with build number generation). Can you please update with the latest Build.ps1 from the main Serilog repo?